packages/pouchdb-replication/src/replicate.js

```diff
@@ -73,15 +74,17 @@ function replicate(src, target, opts, returnValue, result) {
           errorsById[res.id] = res;
```

Comment:
I'm not sure if I understand the context here: for me iterating twice (first over `res`, and then `docs`) is redundant, as it makes sense only for case when `res.length !== docs.length`. On the other hand, in case when `docs.length > res.length`, adding extra docs to `changedDocs` is questionable.